applications/RomApplication/python_scripts/calculate_rom_basis_output_process.py

```diff
@@ -58,10 +58,9 @@ def __init__(self, model, settings):
 
         # Set the ROM basis output settings
         self.rom_basis_output_format = settings["rom_basis_output_format"].GetString()
-        if not self.rom_basis_output_format == "json":
-            err_msg = "Provided \'rom_basis_output_format\' is {}. Available options are \'json\'.".format(self.rom_basis_output_format)
+        if not self.rom_basis_output_format == "json" and not self.rom_basis_output_format == "numpy" :
```

Comment:
```suggestion
        rom_basis_output_available_formats = ["json", "numpy"]
        if self.rom_basis_output_format not in rom_basis_output_available_formats:
```
Just in case we add more in the future.